src/util/transitTypeHelper.tsx

```diff
@@ -0,0 +1,20 @@
+import TransitType from '../enums/transitType';
+
+export default class TransitTypeHelper {
+    public static convertTransitTypeCodeToTransitType = (type: string) => {
```

Comment:
Is this helper really needed? Can we capsule this inside factories?

Reply:
Currently we also use it outside of the factory, so in that sense we need it. This however relates to if NetworkLayers.tsx logic should be in factory or not, which is mentioned in https://github.com/HSLdevcom/jore-map-ui/pull/234#discussion_r221507893